tests/ext-popup.test.js

```diff
@@ -0,0 +1,48 @@
+import Popup from '../src/lib/ext-popup';
+import * as ExtListen from '../src/lib/ext-listen';
+
+test('test init popup when no extension is being monitored', async () => {
+  document.body.innerHTML =
+    '<div>' +
+    '  <p id="monitorStatus"></p>' +
+    '  <button id="startMonitorBtn"></button>' +
+    '  <button id="stopMonitorBtn"></button>' +
+    '  <button id="actLogPage"></button>' +
+    '  <p id="errorText"></p>' +
+    '</div>';
```

Comment:
We could actually load the real document.body from the popup.html file, which would be better for at least a couple of reasons:
- the test would be also make sure that popup.html is what ext-popup.js expects
- we don't have to keep both the test and the popup.html file in sync by manually copying changes between them

To achieve that we could use:
- the nodejs "path" and "fs" modules, to be able to load the file content using something like:
```
  const popupHtml = fs.readFileSync(
    path.resolve(__dirname, "../src/popup/popup.html"),
    "utf-8"
  );
```
- then parse the content of the popup.html file as an HTML document, by creating an instance of [DOMParser](https://developer.mozilla.org/en-US/docs/Web/API/DOMParser) (which is available in jest thanks to JSDOM, available as window.DOMParser) and then using the parseFromString method, it would look like more or less as in: ```domParser.parseFromString(popupHtml, "text/html").body.innerHTML```

(it may be reasonable to do it once outside of the test functions and set it on a `const popupBody`, to eventually reuse it for multiple tests).

Reply:
Thanks for the explanation. I have updated this part using "path" and "fs" module.